main.tf

```diff
@@ -28,7 +28,7 @@ locals {
 }
 
 resource "ibm_scc_provider_type_instance" "scc_provider_type_instance_instance" {
-  depends_on       = [time_sleep.wait_for_authorization_policy]
+  depends_on       = [time_sleep.wait_for_scc_wp_authorization_policy]
```

Comment:
if this depends on the time_sleep which depends on the auth policy then I expect to see those resource blocks above this one in the code. Like I said try to keep the dependency flow the same in the code as the dependncy tree order itself.

Reply:
This should actually also depend_on `ibm_scc_instance_settings.scc_instance_settings` because the COS integration must be done before you can use the SCC instance (which means the `ibm_scc_instance_settings` resource block should also be above this in the code)